lib/index.js

```diff
@@ -58,6 +61,10 @@
             }.bind(this));
         }
 
+        var timeout = parseInt(this._options.timeout, 10) || 10000;
+        if (timeout < 0) timeout = 10000;
```

Comment:
10s seems like a good default, but due to international mobile users i want to increase to 30s in daiquiri.